### ISSUE TITLE
Feature request: make notifications so that they don't disappear; support adding specific time for timers

### ISSUE THREAD

**Issue by doctea** [CoreyHayward/PowerToys-Run-Timer#5]
e.g. if I want to be reminded to do something tomorrow but might not be at my PC when the notification appears, I'd like to be able to tell the reminder to appear at "3PM tomorrow" and to remain on screen until I specifically acknowledge it.

**Comment by beatrizq** [comment]
Agree, this would be awesome. The notification disappears too quickly
